### ISSUE TITLE
removing date tags from Calendar

### ISSUE THREAD

**Issue by Christian-Klaperek** [fbgallet/roam-extension-calendar#4]
When blocks are shown in the calendar, all text in the block, along with the date tag, are displayed in the calendat. It makes sense that the date tag would also be shown, because that the reference used to put it in the calendar. However, dusplaying date tags becomes is redundant since it is a calendar, and it is placed on the corresponding date. 

Is there a way to hide the date reference shown in double brackets, so it doesnt show in the calendar?
Also, removing it opens up more space to display other information there.

![image](https://github.com/user-attachments/assets/4d96ea0c-06f9-44eb-bb0d-91a08323c109)

**Comment by fbgallet** [comment]
I agree, i will try to automatically hide date mention in the next version.

**Comment by Christian-Klaperek** [comment]
Wonderful. Thank you so much

**Comment by Christian-Klaperek** [comment]
Hi Fabrice !

I had a few more suggestions, and made a mock-up of them for better
visualization. What do you think?


[image: image.png]
Cheers
Chris

**Comment by fbgallet** [comment]
@Christian-Klaperek , it looks like your image.png has not been uploaded properly, i can't see it.

**Comment by Christian-Klaperek** [comment]
Hi Fabrice

So sorry about that !

I attached it as a jpg. Please let me know if uploads correctly.

Thank you for your excellent extension. It is becoming a new indispensable
part of my workflow   😁

**Comment by Christian-Klaperek** [comment]
Hello Fabrice

Just wondering if the screenshot uploaded or not ....

**Comment by fbgallet** [comment]
Hi,
the screenshot is still not uploaded !

**Comment by Christian-Klaperek** [comment]
Hi Fabrice

So sorry about that.  How about now?

[image: Calendar Suggestions.jpg]

**Comment by fbgallet** [comment]
Still not. Do you see it yourself in Preview ?
The most simple is to paste or directly drop the image in.

**Comment by Christian-Klaperek** [comment]
How about now?

![Calendar Suggestions](https://github.com/user-attachments/assets/4a825e07-a73c-415f-aaa7-ca099d46b144)

**Comment by Christian-Klaperek** [comment]
I just realized, something similar to my first suggestion, (about hiding the date tag in the calendar view). This idea can be applied to the time as well. Could we also remove the display of the time? Becasue we know from the time slots on the left axis, what time it is 

![image](https://github.com/user-attachments/assets/f211ffe1-846f-4703-99f5-0a7ee087d2c2)

**Comment by Christian-Klaperek** [comment]
I also just realized, regarding my second suggestion (regarding the dark theme). In the light theme, the secondary lines at the 30 minute mark, are thinner, making the hour lines stand out nicely. However in the dark theme, all lines come out with the same wieight.  Could you possiblely use a lighter color for the 30 minute lines, as in this mockup?

![image](https://github.com/user-attachments/assets/798a18e0-4577-4277-b35e-454059061720)

**Comment by Christian-Klaperek** [comment]
Hi Fabrice

I didn't hear back, so I just wanted to make sure that the screenshots I
sent made it through on the final time ...

Cheers !
Chris

**Comment by fbgallet** [comment]
Hi Christian,

Yes i've seen the screenshots and noted of your requests ! Sorry for the lack of response, but i haven't took time to work again on Full Calendar yet, because i'm still working on a major update of Live AI Assistant. But it's the next on my list.
I'll keep you posted !

**Comment by Christian-Klaperek** [comment]
Wonderful!  Thank you Fabrice!

**Comment by Christian-Klaperek** [comment]
Hi Fabrice

I found a new issue with the calendar, and thought to let you know.

Issue: After dragging a TODO from the all-day event (at the top), down into
a time slot, the date tag syntax is automatically changed (appending the
time to the date tag).

Roam no longer recognizes the date tag anymore.  As a result, the block
disappears from the Daily Notes page linked reference, and from the
calendar as well.
The attached screenshot shows the issue.

Looking forward to the next Calendar update, after you finish the Live AI
Assistant update 😁

Thank you and kKind regards,
Chris

**Comment by Christian-Klaperek** [comment]
PS.

I just wanted to make it clear.

If I manually input the time, the date tag stays intact, and it shows
correctly in the calendar.

The issue only occurs when dragging the TODO from within the Full Calendar
extension. Perhaps there is a tiny bug somewhere.😁

Kind regards,

Chris

**Comment by fbgallet** [comment]
Hi @Christian-Klaperek ,

I'm finally working on Full Calendar again. I can't reproduce your last issue ("After dragging a TODO from the all-day event (at the top), down into a time slot, the date tag syntax is automatically changed (appending the time to the date tag).")

You haven't joined the screenshot, i'm not sure to get it. Could you show me what is happening ?

**Comment by Christian-Klaperek** [comment]
Hi Fabrice !

Good to hear from you.

I attempted to reproduce the issue just now but was unable to.

That’s good news, though I hope the problem doesn’t recur.


I’ve attached the original screenshots for reference. As you’ll see, when I
saw the problem in February, dragging the item appeared to corrupt the
syntax, which prevented Roam from recognizing it as a valid date.

The second screenshot shows today’s attempt—no issues this time; everything
functions as expected.

Kind regards
Chris

**Comment by Christian-Klaperek** [comment]
Hi Fabrice

Just wondering if you were planning to make any changes to the appearance
based on my earlier suggestions. I’ve revised the suggestions
slightly—please see the attachment.

Kind regards,
Chris

**Comment by Christian-Klaperek** [comment]
Hi Fabrice

Just to expand a bit on point #4 in the attachment from my last email:
Using the 24-hour time format becomes especially helpful when running
queries. I often sort query results by both date and time, and if the time
isn’t in 24-hour format, the results can end up out of chronological order
...

Would it be possible to have the option for 24hr format ?

Kind regards,
Christian

**Comment by Christian-Klaperek** [comment]
Fabrice

Apologies for sending so many emails — I should have compiled everything
into just one mail.  🥴

Just a quick reminder: If it's possible, please don't forget to
automatically hide both the *date* and *time* in all calendar views (month,
week, day, and year), since that information is already visible in the
calendar itself.

Thank you !
Chris

**Comment by Christian-Klaperek** [comment]
Hello Fabrice

Hope you are well !

I just wanted to ask how the next update on the Full Calendar is coming
along?

I am really excited for it 😁

Kind regards
Chris

**Comment by fbgallet** [comment]
Hi Christian,
I'm still working hard on a big update of Live AI extension and I didn’t finish the Full Calendar updates I started last April, which included some of your suggestions. I do plan to get back to it in a few weeks with the goal of adding Google Calendar integration, but I can’t give a precise timeline for now.

Thanks for your support and eagerness to see Full Calendar improve, but I’ve noticed the community seems a bit less enthusiastic about this extension, which so far hasn’t passed the 1,000 downloads in Roam Depot. Unfortunately, that keeps it from being a priority for the moment.

Best regards,
Fabrice

**Comment by Christian-Klaperek** [comment]
Hi Fabrice !

No worries on the timeline - you are very busy, and your many active
developments give us long-time Roam users confidence to stay with Roam for
the long term 😁.

I have an opinion on why downloads haven’t yet reached 1,000, and I offer
it with the utmost respect for your work. Perhaps the reason is that people
aren't finding it as useful as they hoped. And I believe the improvement
points I mentioned may solve that.

To make this clearer, I prepared another mock-up with a before/after
comparison. It shows how just a few small adjustments could greatly
increase usefulness.

If you do make these tweaks and share the update with the community, I
would think you’ll see a substantial rise in downloads.
 Kind regards
Chris

**Comment by Christian-Klaperek** [comment]
Fabrice

Sorry I totally forgot about this other suggestion.
It's self-explanatory, in the attachment.

Cheers
Chris

**Comment by Christian-Klaperek** [comment]
And the 3-day view

(which would show the full text in the block without any truncating for
multiple days)

**Comment by fbgallet** [comment]
Hi Christian,
I've published an update of Full calendar last week, fixing some issues that you have mentioned and adding Google calendar integration. Let me know if it's working better for you.
